clients/pyethereum:develop/pyethapp.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Immediately abort the script on any error encountered
+set -e
+
+if [ $HIVE_TESTNET ];
+then
+  # TODO: Write better sed magic?
+  sed -i "s/  ACCOUNT_INITIAL_NONCE.*/  ACCOUNT_INITIAL_NONCE: 1048576/g" /root/.config/pyethapp/config.yaml
```

Comment:
Ok, removed those lines
Also moved the line that uses jq to process genesis.json above the "set +e" as we want the script to fail if we get any error from that